src/neutral_mixed.cxx

```diff
@@ -363,9 +369,10 @@ void NeutralMixed::finally(const Options& state) {
       // eta_n = (2. / 5) * kappa_n;
       //
 
-      ddt(NVn) += AA * FV::Div_a_Grad_perp((2. / 5) * DnnNn, Vn)    // Perpendicular viscosity
-                + AA * FV::Div_par_K_Grad_par((2. / 5) * DnnNn, Vn) // Parallel viscosity
-        ;
+      ddt(NVn) +=
+          AA * FV::Div_a_Grad_perp((2. / 5) * DnnNn, Vn)      // Perpendicular viscosity
+          + AA * FV::Div_par_K_Grad_par((2. / 5) * DnnNn, Vn) // Parallel viscosity
```

Comment:
Hi @bendudson how come you haven't put limiters on perpendicular viscosity?

Reply:
Perpendicular conduction is also left out. It's true that we know that conduction helps and not hinders, but I'm tempted to change it all to upwind to see if it fixes the problem, and only then remove it from viscosity and conduction. What do you think?

Reply:
Also, I noticed that these operators are taking the same global compile time limiter as the other operators. I saw quite a significant impact on the results when I tried MinMod in a test case a few months back. It would probably be useful to have the ability to keep MC for most of the equations and have MinMod or Upwind or similar only for the neutral operators. 
But again, more than happy for everything to be on MinMod for the first test run if it fixes the problem..

Reply:
Hey @mikekryjak . I've started by only putting limiters on advection terms: The perpendicular pressure diffusion is an advection with a velocity proportional to the pressure gradient, not a diffusion. I've left the diffusion terms as center differences for now (viscosity and heat conduction).

Reply:
Changing to Upwind (from MC) seems to improve the result (and speed it up). Still failing the integrated test for some reason.